packages/core/src/lib/lit-core.ts

```diff
@@ -117,15 +118,6 @@ const NETWORKS_REQUIRING_SEV: string[] = [
   LitNetwork.DatilTest,
 ];
 
-// The only network we consider entirely static, and thus ignore EPOCH changes for, is Cayenne
```

Comment:
Replaced with `CENTRALISATION_BY_NETWORK` since that is what we really needed here, and Anson added it earlier this week 🎉  💯